### ISSUE TITLE
run ansible control image with --volumes-from

### ISSUE THREAD

**Issue by sometheycallme** [cleanerbot/ansible-security#24]
Double check tests for the playbooks-data created volume (/etc/ansible) being available for ansible controller image

So far so good.

```
ubuntu@box158:~$ docker create --name playbooks-data playbooks-data true
WARNING: Your kernel does not support memory swappiness capabilities, memory swappiness discarded.
3a6672029a3d1ae70daad109ef063bbd3fbc3d65604c5ddadbb2e33b1b9225d7

ubuntu@box158:~$ docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             VIRTUAL SIZE
ansible-security    latest              e5175ed40b22        2 minutes ago       81.24 MB
playbooks-data      latest              3b3121d0d3e1        2 minutes ago       201.5 kB
alpine              3.3                 0a3b5ba3277d        7 days ago          4.793 MB
ubuntu@box158:~$ 
ubuntu@box158:~$ 
ubuntu@box158:~$ docker run -d --volumes-from playbooks-data -t -i --entrypoint bash ansible-security:latest
WARNING: Your kernel does not support memory swappiness capabilities, memory swappiness discarded.
cbe7a22761980ec4f70fcc469ef2cf8b49b32a2691d7cdb2d520d4afa3d898c9

```

**Comment by sometheycallme** [comment]
```
ubuntu@box1571:~/ansible-security$ docker run --volumes-from playbooks-data -t -i --entrypoint bash ansible-security:latest -c "cd /opt/ansible; ansible --version"
WARNING: Your kernel does not support memory swappiness capabilities, memory swappiness discarded.
ansible 2.0.1.0
  config file = 
  configured module search path = Default w/o overrides
```

**Comment by sometheycallme** [comment]
Fixed #25
